src/engine/renderer/tr_shader.cpp

```diff
@@ -2214,26 +2214,8 @@ static bool ParseStage( shaderStage_t *stage, const char **text )
 		}
 		else if ( !Q_stricmp( token, "videoMap" ) )
 		{
-			if ( stage->collapseType != collapseType_t::COLLAPSE_none )
-			{
-				Log::Warn("keyword '%s' cannot be used in collapsed shader '%s'", token, shader.name );
-			}
-
-			token = COM_ParseExt2( text, false );
-
-			if ( !token[ 0 ] )
-			{
-				Log::Warn("missing parameter for 'videoMap' keyword in shader '%s'", shader.name );
-				return false;
-			}
-
-			stage->bundle[ 0 ].videoMapHandle = ri.CIN_PlayCinematic( token, 0, 0, 512, 512, ( CIN_loop | CIN_silent | CIN_shader ) );
-
-			if ( stage->bundle[ 0 ].videoMapHandle != -1 )
-			{
-				stage->bundle[ 0 ].isVideoMap = true;
-				stage->bundle[ 0 ].image[ 0 ] = tr.scratchImage[ stage->bundle[ 0 ].videoMapHandle ];
-			}
+			Log::Warn("videoMap unsupported");
```

Comment:
Can you put a comment explaining a bit why this was removed, with a link to this PR?

Reply:
Added a link here. I put a description of why it is done in the commit message.

Reply:
Thank you very much.